src/app/components/annotations/components/annotation-search-form/annotation-search-form.component.html

```diff
@@ -196,3 +231,39 @@
 <ng-template #tagsTypeaheadTemplate let-result="result" let-searchTerm="term">
   <ngb-highlight [result]="result.text" [term]="searchTerm" />
 </ng-template>
+
+<ng-template
+  #importFileTypeaheadTemplate
+  let-result="result"
+  let-searchTerm="term"
+>
+  <ngb-highlight
+    [result]="result.name"
+    [term]="searchTerm"
+    [ngbTooltip]="importFileTooltip"
+    [tooltipContext]="{ result }"
+  />
+</ng-template>
+
+<!--
+  Users have expressed that it is common to have files with overlapping,
+  duplicate, or not very friendly file names and that other context such as
+  the created date would help differentiate files.
+  To prevent UI clutter (by showing all the information all the time), we
+  show extra information in a tooltip that appears when hovering over the
+  file name.
+-->
+<ng-template #importFileTooltip let-result="result">
+  <div class="import-file-tooltip">
+    <strong>{{ result.name }}</strong>
+
+    <hr />
+
+    <ul class="data-list">
+      <li>
```

Comment:
we could also add a link to the source object

Reply:
We don't currently have a details page for the annotation import files.

We can easily add a download link?

Or do you think that a rich details page (like the audio recording list page) is helpful?

Reply:
Discussed offline: Adding a tooltip that stays open on hover is quite hard without some arch changes that are out of scope for this PR

Additionally, we thought of having an "info" template that can be provided to the typeahead input which would show a small "info" icon that could be toggled to show additional info.

Related issue: https://github.com/QutEcoacoustics/workbench-client/issues/2567